app/elements/submission/sequential/sequential-challenge.html

```diff
@@ -43,8 +43,8 @@
 			margin: 0;
 			padding-top: 10px;
 		}
-		#leftFill{
-			width: 50px;
+		#leftProgressFlex{
+			margin-right: 50px;
```

Comment:
This is specific to sequential challenges and not reused anywhere. If you want to code on your phone, please die.